### PR TITLE
Initial SplitItem + CheckableItem support

### PR DESCRIPTION
**Pull request by jtreminio**
Not yet ready to merge, pending unit tests and feedback.

### CODE REVIEW

File: src/MenuItem/SplitItem.php [hunk 1]
```diff
@@ -129,8 +129,18 @@ public function getRows(MenuStyle $style, bool $selected = false) : array
         return $this->buildRows(
             array_map(function ($index, $item) use ($selected, $length, $style) {
                 $isSelected = $selected && $index === $this->selectedItemIndex;
+
+                if (is_a($item, CheckableItem::class)) {
```
Comment (AydinHassan) [inline src/MenuItem/SplitItem.php]: Any reason for `is_a` over `instanceof` ?

**Comment by AydinHassan** [comment]
Looks good to me - please add some tests and documentation 👍

File: src/MenuItem/SplitItem.php (hunk 1 above)
Comment (jtreminio) [reply src/MenuItem/SplitItem.php]: Old habit I'm trying to break. Will use `instanceof` instead.

**Comment by AydinHassan** [comment]
Thank you @jtreminio !

**Pull request merged**